src/Settings.cpp

```diff
@@ -140,9 +146,13 @@ void Settings::parseInput(int argc, char** argv)
         {"remotename", required_argument, NULL, 'K'},  // Client name on hub server
         {"appendthreadid", no_argument, NULL,
          OPT_APPENDTHREADID},                        // Append thread id to client names
+#ifdef __RT_AUDIO__
         {"rtaudio", no_argument, NULL, 'R'},         // Run in JamLink mode
         {"srate", required_argument, NULL, 'T'},     // Set Sample Rate
-        {"deviceid", required_argument, NULL, 'd'},  // Set RTAudio device id to use
+        {"deviceid", required_argument, NULL, 'd'},  // Set RTAudio device id to use (DEPRECATED)
+        {"audiodevice", required_argument, NULL, OPT_AUDIODEVICE},  // Set RTAudio devices by name
+        {"listdevices", no_argument, NULL, OPT_LISTDEVICES},  // Set RTAudio device id to use
+#endif
         {"bufsize", required_argument, NULL, 'F'},   // Set buffer Size
```

Comment:
Is buffer size also RtAudio-only like sample rate?

Reply:
Ah yes, I think so, I'll change this.
Just FYI while I tried to guard more of the RtAudio code, there might still be some RtAudio stuff in AudioInterface that's not guarded, we might want to try cleaning that up in the future.